Tools/Pipeline/Windows/MainView.cs

```diff
@@ -901,5 +901,12 @@ public void ItemExistanceChanged(IProjectItem item)
                 }
             }
         }
+
+        public bool CopyOrLink(string file, bool exists, out int def, out bool applyforall)
+        {
+            def = 2;
+            applyforall = true;
+            return true;
```

Comment:
@cra0zy - So this is where someone would hook in the dialog for Windows?

What is `def`?  Can you rename that to something more obvious?